test/common/common.go

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
would prefer not to use the "common" folder name, as typically ends up the dumping grounds for everything.